setup/azurecmdfiles.wxi

```diff
@@ -796,6 +786,52 @@
                             <File Id="filAE60D28516977FE205BBEB3D15EB53DE" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.KeyVault\System.Spatial.dll" />
                         </Component>
                     </Directory>
+                    <Directory Id="dirB5DCC9FCC8E6FE17ED06FFC44C161B04" Name="AzureRM.Kubernetes">
```

Comment:
It looks like your repo was not clean prior to generating the installer. I'd recommend cleaning your local repo, rebuilding, and then generating the installer. After doing that, I'd be surprised if this file changes at all since you don't add/remove files or dependencies. See here: https://github.com/Azure/azure-powershell/wiki/Azure-Powershell-Developer-Guide#updating-the-installer

Reply:
Ohh, I cleaned the repo and regenerated the installer but the installer is not showing any changes so unable to push.

Reply:
Please revert the changes to the .wxi file. You won't need to commit this file for this PR.